src/app/components/forms/select-box/terra-select-box.component.ts

```diff
@@ -284,4 +291,105 @@ export class TerraSelectBoxComponent implements OnInit, OnChanges
             }
         }
     }
+
+    protected onKeyDown(event:KeyboardEvent):void
```

Comment:
Bitte nicht kopieren.. Stattdessen eine gemeinsame Basisklasse einführen.

Reply:
na gut... hatte ich vor, wollte aber nicht beide inputs testen...

Reply:
kann man das nicht in ner extra-story aufnehmen?

1. wäre es viel Umbau + Testing und 
2. kann man dann (fast) alles in ne Base-Klasse auslagern

Reply:
Es ist eigentlich kaum Arbeit. Du musst nur die entsprechende Funktionalität in die Basisklasse verschieben. Und nur das was du hier brauchst. Alles andere kann dann später successive in die Basisklasse verschoben werden.

Reply:
Solange du an dem Code dann nichts veränderst, musst du auch nichts großartiges testen

Reply:
-.- na gut